guides/common/modules/con_uefi-http-booting-requirements-with-managed-dhcp.adoc

```diff
@@ -1,25 +1,26 @@
 :_mod-docs-content-type: CONCEPT
 
-[id="http-booting-requirements-with-managed-dhcp"]
-= HTTP booting requirements with managed DHCP
+[id="uefi-http-booting-requirements-with-managed-dhcp"]
+= UEFI HTTP booting requirements with managed DHCP
 
-To provision machines through HTTP booting ensure that you meet the following requirements:
+To provision machines through UEFI HTTP booting ensure that you meet the following requirements:
 
 .Client requirements
-For HTTP booting to work, ensure that your environment has the following client-side configurations:
+For UEFI HTTP booting to work, ensure that your environment has the following client-side configurations:
 
+* Your client is capable of booting HTTP(S) over its network interface.
```

Comment:
out of scope but would be a nice follow-up: reword "client" to "host". Also applies to line 14 and 15.